packages/game/src/lib/is-board-empty.test.ts

```diff
@@ -15,9 +15,9 @@ describe('isBoardEmpty', () => {
 
   it('should return false if any cell is not null or undefined', () => {
     const board: Board = [
-      ['one:red bat', 'one:fungar', 'one:geezard'],
-      ['one:red bat', 'one:fungar', 'one:geezard'],
-      ['one:red bat', 'one:fungar', undefined],
+      ['one:one:red bat', 'one:one:fungar', 'one:one:geezard'],
+      ['one:one:red bat', 'one:one:fungar', 'one:one:geezard'],
+      ['one:one:red bat', 'one:one:fungar', undefined],
```

Comment:
Could we use the `createBoardFromHand` helper here to simplify.

Reply:
Updated helper to take a Hand instead of a Card[], updated calls where necessary.